ChineseFilter/sections/specific.txt

```diff
@@ -2,6 +2,7 @@
 ! Section contains rules for specific websites
 !
 ||mypianku.net/ajax/MyJs/
+||soxscc.net/wsc/los.*.js
```

Comment:
Are you sure that this rule is necessary for desktop? I can see it only on `https://m.soxscc.net/ZheTian/396421.html`, so the rule should be `||m.soxscc.net/wsc/los.*.js` and moved to Mobile filter.